test/jdk/java/awt/Dialog/ChoiceModalDialogTest.java

```diff
@@ -0,0 +1,138 @@
+            r = new Robot();
+            r.setAutoDelay(100);
+            EventQueue.invokeAndWait(() -> {
+                f = new Frame("Frame");
+                c = new Choice();
+                f.setBounds(100, 300, 300, 200);
+                f.setLayout(new FlowLayout());
+                tf = new TextField(3);
+                f.add(tf);
+
+                c.add("1");
+                c.add("2");
+                c.add("3");
+                c.add("4");
+                f.add(c);
+
+                tf.addFocusListener(new FocusAdapter() {
+                    public void focusLost(FocusEvent ev) {
+                        d = new Dialog(f, "Dialog", true);
+                        d.setBounds(300, 300, 200, 150);
+                        d.addKeyListener(new KeyAdapter() {
+                            public void keyPressed(KeyEvent ev) {
+                                keyOK = true;
+                            }
+                        });
+                        d.addMouseListener(new MouseAdapter() {
+                            public void mousePressed(MouseEvent ev) {
+                                mouseOK = true;
+                            }
+                        });
+                        d.setVisible(true);
+                    }
+                });
+
+                f.setVisible(true);
+                f.toFront();
+            });
+            r.delay(1000);
+            r.waitForIdle();
+            EventQueue.invokeAndWait(() -> {
+                r.mouseMove(tf.getLocationOnScreen().x + tf.getSize().width / 2,
+                        tf.getLocationOnScreen().y + tf.getSize().height / 2);
+            });
+            r.delay(500);
+            r.waitForIdle();
+            EventQueue.invokeAndWait(() -> {
+                r.mouseMove(c.getLocationOnScreen().x + c.getSize().width - 4,
+                        c.getLocationOnScreen().y + c.getSize().height / 2);
+                r.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                r.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+            });
+            r.delay(500);
+            r.waitForIdle();
+            EventQueue.invokeAndWait(() -> {
+                r.mouseMove(d.getLocationOnScreen().x + d.getSize().width / 2,
+                        d.getLocationOnScreen().y + d.getSize().height / 2);
+                r.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                r.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+                r.keyPress(KeyEvent.VK_A);
+                r.keyRelease(KeyEvent.VK_A);
+
+            });
+        } finally {
+            EventQueue.invokeAndWait(() -> {
+                if (d != null) {
+                    d.dispose();
+                }
+                if (f != null) {
+                    f.dispose();
+                }
+            });
+        }
+
+        if (!mouseOK || !keyOK) {
+            throw new RuntimeException("Test FAILED");
+        }
+
+        System.out.println("Test Passed!");
```

Comment:
System.out.println("Test Passed!"); is redudant and can be removed.

Reply:
It will useful for stdout print apart from jtreg output.